### ISSUE TITLE
Focus isn't wrapping when tabbing (only works with shift-tab)

### ISSUE THREAD

**Issue by robdodson** [PolymerElements/iron-overlay-behavior#170]
### Description
Tabbing through a modal paper-dialog does not trap focus. However, if I shift-tab it does.

### Expected outcome

Focus should wrap, if i'm tabbing forward

### Actual outcome

Focus escapes the modal when tabbing forward

### Live Demo
http://jsbin.com/ciqugegulu/1/edit?html,output

Note that autofocus also seems messed up in jsbin but is working for me locally

### Steps to reproduce

Click the open modal button
Click inside the modal to move focus navigation start point
Press tab 3 times and note it does not go back to `decline` button.

### Browsers Affected

- [ x] Chrome
- [ ] Firefox
- [ ] Safari 9
- [ ] Safari 8
- [ ] Safari 7
- [ ] Edge
- [ ] IE 11
- [ ] IE 10

**Comment by valdrinkoshi** [comment]
Hi @robdodson, you should test jsbin in output mode only, otherwise the edit iframe won't allow to block the focus.
I cannot reproduce the issue on chrome mac (me opening, clicking in the dialog, then hitting tab):
![dialog-focus](https://cloud.githubusercontent.com/assets/6173664/15121480/bb450438-15cf-11e6-89aa-44d91f7be19e.gif)

**Comment by robdodson** [comment]
Hm I was actually seeing the problem in a local version and I made the jsbin to reflect what I saw. Let me check the version my app is using

**Comment by robdodson** [comment]
Here it is in my app:

![dialog](https://cloud.githubusercontent.com/assets/1066253/15126875/1307fc34-15e9-11e6-9544-c0ad5a34d582.gif)

When it opens I hit shift-tab (which does properly trap focus although it causes the page to scroll which is weird). Then I hit tab and it escapes and focus moves into my browser tabs.

This is paper-dialog v1.0.4 in Shrine app. My markup looks like this:

```
<body unresolved>

  <template is="dom-bind">

    <iron-ajax url="data/items.json" auto last-response="{{items}}"></iron-ajax>
    <iron-ajax url="data/featured.json" auto last-response="{{featuredItems}}"></iron-ajax>
    <shrine-app items="[[items]]" featured-items="[[featuredItems]]"></shrine-app>

    <paper-dialog modal id="modal">
      <h2>Would you like to save?</h2>
      <p>Otherwise all will be deleted!</p>
      <div class="buttons">
        <button dialog-dismiss style="margin-right: 16px;">Decline</button>
        <button dialog-confirm autofocus>Accept</button>
      </div>
    </paper-dialog>

  </template>

</body>
```

**Comment by valdrinkoshi** [comment]
Unfortunately I can't manage to reproduce it, and I cannot see any error from your markup.
Does that happen also if you remove the shrine-app & add a button to open the dialog?

**Comment by robdodson** [comment]
yep this has the same behavior with 1.0.4

```
<body>

  <button onclick="modal.open();">Open modal</button>

  <paper-dialog modal id="modal">
    <h2>Would you like to save?</h2>
    <p>Otherwise all will be deleted!</p>
    <div class="buttons">
      <button dialog-dismiss style="margin-right: 16px;">Decline</button>
      <button dialog-confirm autofocus>Accept</button>
    </div>
  </paper-dialog>

</body>

````

**Comment by valdrinkoshi** [comment]
Tested it locally, and I can reproduce the issue, in the sense that the focus goes to the search bar.
Will further investigate if it is possible to block this behavior, marking this as a bug for now.

**Comment by robdodson** [comment]
It should be possible to block it. Here's a modal I wrote that does so
https://robdodson.github.io/udacity-a11y/lesson2-focus/07-modals-and-keyboard-traps/index-solution.html

**Comment by valdrinkoshi** [comment]
Yep, it should be doable by updating `_onCaptureTab` method.
Contributions are welcome if you feel like it, @robdodson ;)

**Comment by robdodson** [comment]
I can try to look into it. I'm featuring the dialog in my I/O talk so I'd
love to get it fixed if possible. I'm a little backed up writing the talk
itself so if you have any spare cycles to work on it with me that would be
rad.

**Comment by valdrinkoshi** [comment]
Ok, I'll give it a shot tomorrow ;)

**Comment by valdrinkoshi** [comment]
@robdodson fixed & released in `v1.7.3`
